### ISSUE TITLE
Awesome open geoscience

### ISSUE THREAD

**Issue by MuellerSeb** [psyplot/psyplot#25]
This awesome package should be listed here:
https://github.com/softwareunderground/awesome-open-geoscience

There is a visualization section, where psyplot fits perfectly. :wink: 

You can add it by simply opening a PR.

Cheers, Sebastian

**Comment by Chilipp** [comment]
true! thanks @MuellerSeb for pointing me to that :smiley:  I'll make the PR as soon as I finally made the new release :see_no_evil:
